src/main/kotlin/org/jetbrains/teamcity/rest/implementation.kt

```diff
@@ -66,18 +68,24 @@ internal class TeamCityInstanceImpl(private val serverUrl: String,
 
 private class BuildLocatorImpl(private val service: TeamCityService, private val serverUrl: String): BuildLocator {
     private var buildConfigurationId: BuildConfigurationId? = null
+    private var number: String? = null
     private var status: BuildStatus? = BuildStatus.SUCCESS
     private var tags = ArrayList<String>()
     private var count: Int? = null
     private var branch: String? = null
     private var includeAllBranches = false
     private var pinnedOnly = false
 
-    override fun fromConfiguration(buildConfigurationId: BuildConfigurationId): BuildLocator {
+    override fun fromConfiguration(buildConfigurationId: BuildConfigurationId): BuildLocatorImpl {
```

Comment:
Actually there are no mistakes, it was a socket connection issue, is it possible to restart check?
I've committed some useless comment and then checks have passed - https://travis-ci.org/JetBrains/teamcity-rest-client/builds/243237173